utils/run.js

```diff
@@ -3,8 +3,8 @@ const { join, relative, sep } = require('path');
 
 // The order matters because we must build dependencies first
 const allPackages = [
-  'frameworks',
   'now-routing-utils',
+  'frameworks',
```

Comment:
Why the change of order here?

Reply:
Now that I think about it, maybe frameworks should be built after build-utils so it can import `readConfigFile`

Reply:
Cause `frameworks` requires the `Route` type from `routing-utils` (see src/types.ts).

Reply:
Maybe we should remove `readConfigFile` from build-utils and anything relying on it can import from frameworks instead?

Reply:
That's what this PR is doing 😛